### PR TITLE
4337 Module readme cleanup

### PR DESCRIPTION
**Pull request by mmv08**
This PR:
- Cleans up the readme of the 4337 module
- Fix grammar/spelling errors, rewrite some sentences for better clarity

### CODE REVIEW

File: 4337/README.md [hunk 1]
```diff
@@ -4,7 +4,7 @@
 
 ## Execution Flow
 
-The diagram below outlines the flow that is triggered when a user operation is submitted to the entrypoint. Additionally the gas overhead compared to a native implementation is mentioned.
+The diagram below outlines the flow triggered when a user operation is submitted to the entrypoint. Additionally, the gas overhead compared to a native implementation is mentioned.
```
Comment (nlordell) [inline 4337/README.md:7]: Interesting, the sentence feels weird without the "that is", but I trust Grammarly more than me 😅.

File: 4337/README.md [hunk 2]
```diff
@@ -52,11 +52,11 @@ sequenceDiagram
 
 The gas overhead is based on [EIP-2929](https://eips.ethereum.org/EIPS/eip-2929). It is possible to reduce the gas overhead by using [access lists](https://eips.ethereum.org/EIPS/eip-2930).
 
-Note: The gas overhead is only the very base line using storage access and call costs. As also additional costs occur to handle the storage access and calls the actual gas overhead is higher. The tests indicate an overhead of ~11.6k gas when a fee payment is required and 7.8k gas otherwise. This shows that there is still room for optimization in the contracts used in the tests.
+Note: The gas overhead is only the baseline using storage access and call costs. The actual gas overhead is higher as additional costs occur to handle the storage access and calls. The tests indicate an overhead of ~11.6k gas when a fee payment is required and 7.8k gas otherwise. This shows that there is still room for optimization in the contracts used in the tests.
```
Comment (nlordell) [inline 4337/README.md:55]: Is the 11.6K and 7.8K up to date?

File: 4337/README.md [hunk 3]
```diff
@@ -82,7 +82,7 @@ bytes memory deployData = abi.encodeWithSignature("createProxyWithNonce", SAFE_S
 bytes memory initCode = abi.encodePacked(SAFE_PROXY_FACTORY_ADDRESS, deployData);
 ```
 
-The diagram below outlines the flow that is triggered by the initialization data to deploy a Safe with the 4337 module enabled.
+The diagram below outlines the flow triggered by the initialization data to deploy a Safe with the 4337 module enabled.
```
Comment (nlordell) [inline 4337/README.md:85]: I realised that there is a bug in the sequence diagram 👇 that I missed in #112 that you can ninja edit in this PR: the 4337 is set as the fallback handler **and** enabled as a module (the `EntryPoint` is no longer enabled as a module).

**Review by nlordell — APPROVED** [review]
Some questions, but the changes are good as is.

File: 4337/README.md (hunk 2 above)
Comment (mmv08) [reply 4337/README.md]: Actually they were not, I updated the estimations

**Pull request merged**